addons/im_livechat/static/tests/embed/livechat_service_tests.js

```diff
@@ -19,14 +19,16 @@ QUnit.test("persisted session history", async () => {
     const channelId = pyEnv["discuss.channel"].create({
         channel_member_ids: [
             Command.create({ partner_id: pyEnv.adminPartnerId }),
-            Command.create({ guest_id: guestId }),
+            Command.create({ guest_id: guestId, is_minimized: true }),
         ],
         channel_type: "livechat",
         livechat_channel_id: livechatChannelId,
         livechat_operator_id: pyEnv.adminPartnerId,
     });
-    const [channelInfo] = pyEnv.mockServer._mockDiscussChannelChannelInfo([channelId]);
-    cookie.set("im_livechat_session", JSON.stringify(channelInfo));
+    cookie.set(
+        "im_livechat.saved_state",
+        JSON.stringify({ threadData: { id: channelId, model: "discuss.channel" }, persisted: true })
+    );
```

Comment:
A bit bothering the cookie is forged in the test.

There's ongoing work to improve internals of models and identity of discuss objects will be changed as a result.
By forging the cookie in this test, there's high probability the test pass when in practice it doesn't. Reason being that I might forget to update livechat code and the test pass because it relies on old way of identifying thread.

Probably if I forget to update livechat code I will also forget to update `threadData` parts and without implementing new planned ways to create threads, the old way to identify a thread will be preserved. So all this to say we'll probably have time to adapt this code in test before this becomes an issue!